server/webapp/WEB-INF/rails/webpack/views/components/ellipsize/index.scss

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
`@import "../../global/common";`?

Reply:
Not needed

Reply:
@bdpiparva -- it is required. We've included `typography.scss` and `type.scss` into `global/common.scss`.

`typography.scss` modifies the default styles of the HTML element, eg `p` tag:
```
p {
  margin: 0 0 20px 0;
}
```

So, if we don't include `common.scss` on all the pages, the paragraph on one page might not look the same on the page where it is not included.

This theory applies for a bunch of HTML elements including: `h1`... `h6`.

---

IMO, we can look at it as a practice, regardless of whether it is required for the page or not.

Reply:
I see. I will do that. I was not aware that it is applying `typography.scss` per component. I was under impression that it is applied at the page level.

Reply:
@GaneshSPatil - `typography.scss` is still included in the spa_base. No need to include `common.scss` in every component. Only need to include it we want to use global variables.

There is `type.scss` which does some css styling on the named tag(`p`) in this case.
I will try to move it to `spa_base.scss`.